### ISSUE TITLE
How to disable CSRF token

### ISSUE THREAD

**Issue by danijel2017** [multidots/cakephp-rest-api#31]
Hello 

Now am going to make rest api using your plugin in cake php 3.6.x
But can't test api in post man because csrf token problem.
I think this is disabled in rest api.
Please let me know how to do this.

Thanks

**Comment by yusser** [comment]
Go to your src/Application.php file and comment lines below:
![captura de tela de 2018-09-02 12-00-22](https://user-images.githubusercontent.com/1784213/44957361-dcae9300-aea7-11e8-9541-55f5fde1049a.png)

this will make cakephp ignore csrf protection.

**Comment by justinhartman** [comment]
> Go to your src/Application.php file and comment lines below:
> this will make cakephp ignore csrf protection.

@yusser yes, this solves the problem but this doesn't solve the issue. I don't think that CSRF should be disabled globally and one should be able to use it along with the API but right now, you can't. Is there any way around this? What needs to be fixed and I don't mind taking a look to see if I can fix it in a PR.
